src/material/datepicker/datepicker.md

```diff
@@ -257,6 +257,19 @@ By default the `MomentDateAdapter` will creates dates in your time zone specific
 })
 ```
 
+By default the `MomentDateAdapter` will parse dates in a forgiving way. This may result in dates
+being parsed incorrectly. You can change the default behaviour to parse dates strictly by providing
+the `MAT_MOMENT_DATE_ADAPTER_OPTIONS` and setting it to `strict: true`.
+
+```ts
+@NgModule({
+  imports: [MatDatepickerModule, MatMomentDateModule],
+  providers: [
+    { provide: MAT_MOMENT_DATE_ADAPTER_OPTIONS, useValue: { strict: true } }
```

Comment:
We don't usually put spaces around the braces. It should be like this:

```ts
{provide: MAT_MOMENT_DATE_ADAPTER_OPTIONS, useValue: {strict: true}}
```

Reply:
Those have been there since 9a85b9ba74 but I'll remove them.

Reply:
I've also removed them from the other example.